src/services/identity/SitesCacheService.ts

```diff
@@ -0,0 +1,93 @@
+import _ from "lodash"
+
+import {
+  ISOMERPAGES_REPO_PAGE_COUNT,
+  GH_MAX_REPO_COUNT,
+  ISOMER_ADMIN_REPOS,
+  GITHUB_ORG_REPOS_ENDPOINT,
+} from "@root/constants"
+import { genericGitHubAxiosInstance } from "@root/services/api/AxiosInstance"
+import type { GitHubRepositoryData, RepositoryData } from "@root/types/repoInfo"
+
+// The SitesCacheService is responsible for storing information about
+// sites that is not updated live such as lastUpdated and previewImage.
+// Changes are polled at fixed intervals, stored in cache, and served
+// to avoid long load time from live querying.
+
+const REPO_DATA_REFRESH_INTERVAL = 60 * 1000 // 1 minutes
```

Comment:
can this be injected into the constructor of the service?

Reply:
Okay 👍